git-bash-hints.qmd

```diff
@@ -0,0 +1,188 @@
+---
+title: "Git Bash Hints"
+editor: visual
+---
+
+This page will cover a few useful commands that you can use in Git Bash. 
+
+:::{.callout-note}
+
+You will see many generalised paths, directory names, file names, etc. on this page. For example, you will see something like `cd /path_to/working_directory/you_want` where you would need to replace the `/path_to/working_directory/you_want` part with a path that is applicable to your work.
+
+:::
+
+## Checking and Navigating Directories
+
+The commands in this section allow you to check and change your current working directory, moving between different folders in your file system.
+
+`pwd`: This stands for "print working directory", so it will show you your current working directory for Git.
+
+You can use it by entering the following in the terminal: 
+
+```{r, eval = FALSE}
+
+pwd
+
+```
+
+
+`cd`: This stands for "change directory", so it will point to where you want your directory to be.
+
+To change your working directory, we would amend the code below to reflect the path of the working directory we want and enter it into the terminal: 
+
+```{r, eval = FALSE}
+
+cd /path_to/working_directory/you_want
+
+```
+
+ 
+## Viewing Files and Directories
+
+The commands in this section help you list and view the contents of directories, providing detailed information about files and sub-directories.
+
+`ls`: This stands for "listing".
+
+To list all files or sub-directories, we would type the following in the terminal: 
+
+```{r, eval = FALSE}
+
+ls
+
+```
+
+
+To list all files or sub-directories, including hidden files, we would type the following in the terminal: 
+
+```{r, eval=FALSE}
+
+ls -a
+
+```
+
+
+## Creating and Removing Files and Directories
+
+### Creating directories 
+
+The commands in this section enable you to create new directories, remove files, and delete empty directories.
+
+
+`mkdir`: This stands for "make directory".
+
+To create a directory, we would amend the code below to reflect the path we want and the new directory name and enter it into the terminal: 
+
+```{r, eval=FALSE}
+
+mkdir /path_to_the_directory_you_want_to_create/new_directory_name
```

Comment:
Again worth noting this can be absolute or relative paths

Reply:
In fact I'd simplify the example to just be realtive, so `mkdir new_directory_name` and say that will create a new directory in your current working directory.